ledger-core/application/testutils/launchnet/launchnet.go

```diff
@@ -187,10 +192,6 @@ func GetDiscoveryNodesCount() (int, error) {
 }
 
 func GetNodesCount() (int, error) {
-	if isCloudMode() {
```

Comment:
I think TestGetStatus will fail due to that

Reply:
fixed